src/engine-actions/create-channels.js

```diff
@@ -140,14 +146,22 @@ async function assertBalanceIsSufficient (engine, fundingAmount) {
  *
  * @param {string} paymentChannelNetworkAddress
  * @param {string} fundingAmount - int64 string
- * @param {string} roundBehavior - Behavior when encountering a channel that is too small
+ * @param {object} [options]
+ * @param {string} [options.roundBehavior=CHANNEL_ROUNDING.DOWN] - Behavior when encountering a channel that is too small
+ * @param {number} [options.targetTime=DEFAULT_CONFIRMATION_DELAY] - Estimated time to first confirmation in seconds
```

Comment:
This could be pretty confusing if you don't have the context of fees, `confirmationTargetTimeForFeeRate` is my attempt to capture it in a name, but just adding something about fee targeting to the comment could be good